tests/v0_tests/test_bulk_search.py

```diff
@@ -111,61 +197,37 @@ def test_search_highlights(self):
             search_res = resp['result'][0]
             assert ("_highlights" in search_res["hits"][0]) is expected_highlights_presence
 
-    def test_search_multi(self):
-        self.client.create_index(index_name=self.index_name_1)
-        d1 = {
-                "doc title": "Cool Document 1",
-                "field 1": "some extra info",
-                "_id": "e197e580-0393-4f4e-90e9-8cdf4b17e339"
-            }
-        d2 = {
-                "doc title": "Just Your Average Doc",
-                "field X": "this is a solid doc",
-                "_id": "123456"
-        }
-        res = self.client.index(self.index_name_1).add_documents([
-            d1, d2
-        ])
-
-        if self.IS_MULTI_INSTANCE:
-            self.warm_request(self.client.bulk_search, [{
-                "index": self.index_name_1,
-                "q": "this is a solid doc"
-            }])
-
+    @with_documents(lambda self: {self.index_name_1: [{
+            "doc title": "Cool Document 1",
+            "field 1": "some extra info",
+            "_id": "e197e580-0393-4f4e-90e9-8cdf4b17e339"
+        },{
+            "doc title": "Just Your Average Doc",
+            "field X": "this is a solid doc",
+            "_id": "123456"
+    }]}, warmup_query="this is a solid doc")
+    def test_bulk_search_multi(self, index_1_docs):
         resp = self.client.bulk_search([{
             "index": self.index_name_1,
             "q": "this is a solid doc"
         }])
         assert len(resp['result']) == 1
         search_res = resp['result'][0]
 
-        assert d2 == self.strip_marqo_fields(search_res['hits'][0], strip_id=False)
+        assert index_1_docs[1] == self.strip_marqo_fields(search_res['hits'][0], strip_id=False)
         assert search_res['hits'][0]['_highlights']["field X"] == "this is a solid doc"
 
-    def test_select_lexical(self):
-        self.client.create_index(index_name=self.index_name_1)
-        d1 = {
+    @with_documents(lambda self: {self.index_name_1: [{
             "doc title": "Very heavy, dense metallic lead.",
             "field 1": "some extra info",
             "_id": "e197e580-0393-4f4e-90e9-8cdf4b17e339"
-        }
-        d2 = {
+        },{
             "doc title": "The captain bravely lead her followers into battle."
                          " She directed her soldiers to and fro.",
             "field X": "this is a solid doc",
             "_id": "123456"
-        }
-        res = self.client.index(self.index_name_1).add_documents([
-            d1, d2
-        ])
-
-        if self.IS_MULTI_INSTANCE:
-            self.warm_request(self.client.bulk_search, [{
-                "index": self.index_name_1,
-                "q": "Examples of leadership"
-            }])
-
+        }]}, warmup_query="Examples of leadership")
+    def test_select_lexical(self, index_1_docs):
```

Comment:
This search isn't even lexical (acknowledged that this naming issue was introduced a while ago)

Reply:
It is testing lexical? 
```python
        # try it with lexical search:
        #    can't find the above with synonym
        assert len(self.client.index(self.index_name_1).search(
            "Examples of leadership", search_method=marqo.SearchMethods.LEXICAL)["hits"]) == 0
        #    but can look for a word
        assert self.client.index(self.index_name_1).search(
            '"captain"')["hits"][0]["_id"] == "123456"
```
Essentially checking `search_method=marqo.SearchMethods.LEXICAL` means synonym queries will have zero hits.